edb/repl/__init__.py

```diff
@@ -850,6 +857,10 @@ def _command_errverbose(
             print('> ' + '\n> '.join(srv_tb.strip().split('\n')))
             print()
 
+    def _command_semicolon_hint(self, arg: Optional[str]) -> None:
+        if arg and arg[-1] == ';':
```

Comment:
```suggestion
        if arg.strip() and arg.strip()[-1] == ';':
```

Reply:
The first `arg` mention ensures that `arg` is neither None nor `''`, both of which would cause index errors for `arg[-1]`. I've added the `strip`.